5_databases/week2/homework

```diff
@@ -0,0 +1,145 @@
+INSERT INTO task (title, description, created, updated, due_date, status_id)
+VALUES ('Learn an  instrument', 'Maybe violin', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP, '2023-12-31', 1);
+
+
+UPDATE task
+SET title = 'Learn an game'
+WHERE title = 'Learn an  instrument';
+
+
+
+UPDATE task
+SET due_date = '2023-01-01 00:00:00'
+WHERE id = 1;
+
+
+
+UPDATE task
+SET status_id = '2'
+WHERE id = "1";
+
+
+UPDATE task
+SET status_id = (SELECT id FROM status WHERE name = 'Done')
+WHERE id = 5;
+
+DELETE FROM task
+WHERE id = 18;
+
+--Part 2: School database
+ -- Create a new database
+CREATE DATABASE IF NOT EXISTS school_database;
+USE school_database;
+
+-- Create the Class table
+CREATE TABLE Class (
+  id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  begins DATE NOT NULL,
+  ends DATE NOT NULL,
+  status ENUM('not-started', 'ongoing', 'finished') NOT NULL
+);
+
+-- Create the Student table
+CREATE TABLE Student (
+  id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  email VARCHAR(255) NOT NULL,
+  phone VARCHAR(255),
+  class_id INT UNSIGNED,
+  FOREIGN KEY (class_id) REFERENCES Class(id) ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+-- Create an index on the name column of the Student table
+CREATE INDEX idx_name ON Student (name);
+
+
+--Part 3: More queries
+-- Get all tasks assigned to users whose email ends in @spotify.com
+
+SELECT *
+FROM task JOIN `user` on email LIKE "%@spotify.com"
```

Comment:
Here you should use the user_task many-to-many table to make relations and same for the next two queries